sabitou_clients/lib/screens/connection/login.dart

```diff
@@ -0,0 +1,191 @@
+              ],
+            ),
+          ),
+          Flexible(
+            child: SingleChildScrollView(
+              child: Container(
+                padding:
+                    const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+                width: 500,
+                child: Column(
+                  children: [
+                    SizedBox(
+                      width: 100,
+                      height: 100,
+                      child: Image.asset('assets/images/logo.png'),
+                    ),
+                    const Center(
+                      child: Text(
+                        'Log in to your account',
+                        style: TextStyle(
+                          fontSize: 19,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                    ),
+                    const SizedBox(height: 5),
+                    const Center(
+                      child: Text(
+                        'Welcome back! Please enter your details.',
+                        style: TextStyle(
+                          fontSize: 16,
+                          color: Colors.grey,
+                        ),
+                      ),
+                    ),
+                    Form(
+                      key: formKey,
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          const Row(
+                            children: [
+                              Text('Email'),
+                              Text('*', style: TextStyle(color: Colors.red)),
+                            ],
+                          ),
+                          TextFormField(
+                            controller: emailController,
+                            decoration: const InputDecoration(
+                              hintText: 'Enter your email',
+                            ),
+                          ),
+                          const SizedBox(height: 15),
+                          const Row(
+                            children: [
+                              Text('Password'),
+                              Text('*', style: TextStyle(color: Colors.red)),
+                            ],
+                          ),
+                          TextFormField(
+                            controller: passwordController,
+                            obscureText: !_isPasswordVisible,
+                            decoration: InputDecoration(
+                              hintText: '**********',
+                              suffixIcon: IconButton(
+                                icon: Icon(
+                                  _isPasswordVisible
+                                      ? Icons.visibility
+                                      : Icons.visibility_off,
+                                ),
+                                onPressed: () {
```

Comment:
Verify and fix the linter issue here
`Prefer extracting the callback to a separate widget method.`